examples/A2A/math_assistant_a2a/src/nat_math_assistant_a2a/register.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections.abc import AsyncGenerator
+from typing import Any
+
+from pydantic import Field
+
+from nat.builder.builder import Builder
+from nat.builder.function import FunctionGroup
+from nat.cli.register_workflow import register_function_group
+from nat.data_models.function import FunctionGroupBaseConfig
+
+
+class LogicEvaluatorConfig(FunctionGroupBaseConfig, name="logic_evaluator"):
+    include: list[str] = Field(default_factory=lambda: ["if_then_else", "evaluate_condition"],
```

Comment:
Not sure if this is too obvious, but it took me a while to figure out what was going on here.

```suggestion
    # Using a lambda so that each config instance receives a unique include list
    include: list[str] = Field(default_factory=lambda: ["if_then_else", "evaluate_condition"],
```

Reply:
This is a relatively common pattern for default_factory specs within Pydantic Fields.